### ISSUE TITLE
plugins not loaded when setMergeCompiledIncludes is true

### ISSUE THREAD

**Issue by artcs** [smarty-php/smarty#435]
With 3.1.32 sometimes modifier plugins are not loaded when `setMergeCompiledIncludes` is set to true:

```
[25-Apr-2018 12:28:46 Europe/Berlin] PHP Fatal error: Call to undefined function smarty_modifier_truncate() in /pathto/var/smarty/tc/af65f1f22c72d1d1f57baa3e9fd13ad52f24362b_1.file.fdview.tpl.php on line 162
```

After deleting compiled templates the first call works fine, subsequent calls fail. Caching is off.
As a workaround if I set `setMergeCompiledIncludes` to false, it works as expected.
Issue started with dev45-version from March 28th.

Testcase:
```
index.tpl:

{include file="included.tpl"}
bla bla
```

```
included.tpl:

{"Something"|truncate:7:'...'}
```

```
index.php:

[...]
$Smarty->setMergeCompiledIncludes(true);
$Smarty->display('index.tpl');
```

**Comment by thedotedge** [comment]
I can reproduce with 3.1.32 if `setMergeCompiledIncludes(true)` is called.

**Comment by uwetews** [comment]
This bug is now fixed in the master branch and will later be included in 3.1.33
